jenkins/jobs/dev_env_integration_tests.pipeline

```diff
@@ -0,0 +1,118 @@
+import java.text.SimpleDateFormat
+
+ci_git_credential_id = "metal3-jenkins-github-token"
+
+// 10 minutes
+def CLEAN_TIMEOUT = 600
+// 2 hours
+def TIMEOUT = 7200
+
+script {
+  if ("${PROJECT_REPO_ORG}" == "metal3-io" && "${PROJECT_REPO_NAME}" == "project-infra") {
+    echo "Checkout ${ghprbAuthorRepoGitUrl} branch ${ghprbActualCommit}"
+    ci_git_branch="${ghprbActualCommit}"
+    ci_git_url = "${ghprbAuthorRepoGitUrl}"
+  } else {
+    echo "Checkout ${ghprbAuthorRepoGitUrl} main"
+    ci_git_branch = "main"
+    ci_git_url = "https://github.com/metal3-io/project-infra.git"
+  }
+
+  def date = new Date()
+  def dateFormat = new SimpleDateFormat("yyyyMMddHHmmss")
+  def rand = new Random()
+  VM_KEY = (1..4).collect {
+    ('a'..
+      'z').join("")[rand.nextInt(26)]
+  }.join("")ect { ('a'..'z').join("")[ rand.nextInt( 26 ) ] }.join("")
```

Comment:
```suggestion
  }.join("")ect { ('a'..'z').join("")[ rand.nextInt( 26 ) ] }.join("")
```
You have a type here. 

Also, we can remove the following completely. We are actually not using VM_KEY variable anymore. 
```suggestion
  }.join("")ect { ('a'..'z').join("")[ rand.nextInt( 26 ) ] }.join("")
```